server/engine/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolPreprocessor.g4

```diff
@@ -119,7 +119,7 @@ openingPseudoTextDelimiter
    ;
 
 closingPseudoTextDelimiter
-   : DOUBLEEQUALCHAR (COMMACHAR | DOT_FS | SEMICOLON_FS)?
```

Comment:
why `DOT_FS` is not a delimiter anymore?

Reply:
https://www.ibm.com/docs/en/cobol-zos/6.3?topic=structure-separators

PseudoTextDelimiter  has no concept of delimiter. We did it to ease the grammar writing (this is what I assumed).
Since DOT_FS is part of the statement delimiter, it results in a wrong tree.

<html>
<body>
<!--StartFragment-->

== | Pseudo-text delimiter
-- | --


<!--EndFragment-->
</body>
</html>

Reply:
Also, as per https://www.ibm.com/docs/en/cobol-zos/6.3?topic=separators-rules

```The separator period must be used only to indicate the end of a sentence, or as shown in formats. The separator comma and separator semicolon can be used anywhere the separator space is used.```

And dot_fs is taken care by sentence syntax grammar